app/src/main/kotlin/com/br/myfavoritehero/util/Constants.kt

```diff
@@ -35,5 +35,7 @@ object Constants {
 
     const val ERROR_NOT_CONNECTED = 321
     const val MESSAGE_ERROR_NOT_CONNECTED = "Parece que você não tem uma conexão estável com a internet."
+    const val MESSAGE_ERROR_TIME_OUT = "Não estamos conseguindo nos conectar com a Marvel, acho que os vingadores estão com problemas! Só voltaremos mais tarde! O que você pode fazer para salvar o mundo enquanto isso?"
```

Comment:
This is better in Strings file in case of future translations.. :D

Reply:
it's already there, but this one is needed to not generate a linkage to the application context on BaseViewModel. Should be a better way to solve this, but for now its ok.